src/pages/userManagement/UserManagerCollaborationEditTable.jsx

```diff
@@ -182,30 +215,48 @@ export default function UserManagersCollaborationEditTable({
           style={{ margin: '8px 16px', display: 'block' }}
         />
       ) : null}
-      {patchError && (
+      {revokeError && (
         <CustomAlert
           severity="error"
           titleId="COLLABORATION_REVOKE_ERROR"
-          onClose={() => {
-            onClearPatchError();
-          }}
+          onClose={onClearRevokeError}
         >
-          {patchError
-            ? patchError +
+          {revokeError
+            ? revokeError +
```

Comment:
I know that this logic wasn't introduced here, but, given the condition for this alert to render at all, isn't this redundant? I don't see how the unknown error message will ever be used.

Reply:
Resolved.